### DISCUSSION TITLE
Default value for video reading

### DISCUSSION THREAD

**Discussion by tchantches007** [openbookings26/ContaCam#28] [Q&A] [answered]
Hello,
Is there a way of setting the default video playback speed to any value greater than 1?

**Comment by openbookings26** [comment, marked as answer]
No natively. What I have is a browser extension like '[Video Speed Controller](https://chromewebstore.google.com/detail/video-speed-controller/nffaoalbilbmmfgbnbgppjihopabppdk)' (there are many others) that let's me modify the playback speed of almost any browser video player (Youtube, Instagram, etc) and it also works with ContaCam. I have mine set to default to 2x so it always plays at that speed.
